monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -0,0 +1,479 @@
+            pt = Point(transformed_x, transformed_y)
+            # build a buffer so that the radius is 70m for the track
+            buffer = pt.buffer(70)
+            buffer_points = zip(buffer.exterior.coords.xy[0], buffer.exterior.coords.xy[1])            
+            proj_buffer_points = []
+            # reproject back to ESPG 4326
+            transformer2 = Transformer.from_crs("epsg:3857", "epsg:4326")
+            for point in buffer_points:
+                x = point[0]
+                y = point[1]
+                x, y = transformer2.transform(x, y)
+                proj_buffer_points.append((x, y))
+            buffered_path = Polygon(proj_buffer_points)
+
+            altitude = altitude_of_ground_level_wgs_84 + self.altitude_agl  # meters WGS 84
+            flight_points_with_altitude = []
+            x, y = buffered_path.exterior.coords.xy
+
+            adjecent_points = [Point(x[0], y[0]), Point(x[1], y[1])]
+            number_of_points = len(proj_buffer_points)
+
+            flight_speed = self.generate_flight_speed(adjecent_points=adjecent_points, number_of_points= number_of_points)
+
+            for coord in range(0, len(x)):
+                flight_points_with_altitude.append(FlightPoint(lat=y[coord], lng=x[coord], alt=altitude))
+
+
+            all_grid_cell_tracks.append(GridCellFlight(bounds=grid_cell, track=flight_points_with_altitude, flight_speed = flight_speed))
+
+        self.grid_cells_flight_tracks = all_grid_cell_tracks
+
+    def make_json_compatible(self, struct: Any) -> Any:
+        if isinstance(struct, tuple) and hasattr(struct, '_asdict'):
+            return {k: self.make_json_compatible(v) for k, v in struct._asdict().items()}
+        elif isinstance(struct, dict):
+            return {k: self.make_json_compatible(v) for k, v in struct.items()}
+        elif isinstance(struct, str):
+            return struct
+        try:
+            return [self.make_json_compatible(v) for v in struct]
+        except TypeError:
+            return struct
+
+    def generate_rid_state(self, duration=180):
+        '''
+
+        This method generates rid_state objects that can be submitted as flight telemetry
+
+
+        '''
+        all_flight_telemetry = {}
+        flight_track_details = {}  # Develop a index of flight length and their index
+        # Store where on the track the current index is, since the tracks are circular, once the end of the track is reached, the index is reset to 0 to indicate beginning of the track again.
+        flight_current_index = {}
+        # Get the number of flights
+        num_flights = len(self.grid_cells_flight_tracks)
+
+        now = arrow.now()
+        now_isoformat = now.isoformat()
+        for i in range(num_flights):
+            flight_positions_len = len(self.grid_cells_flight_tracks[i].track)
+            # in a circular flight pattern increment direction
+            angle_increment = int(360 / flight_positions_len)
```

Comment:
`track` is a float; no need to reduce precision to an int, especially when computing a delta whose error will be magnified by the number of points it is multiplied over

Reply:
Keeping precision as float, removing it